src/content/docs/new-relic-solutions/increase-user-engagement.mdx

```diff
@@ -0,0 +1,196 @@
+
+Our website performance monitoring (WPM) tool gives you insight about your site's availability and performance. With out of the box alerts and routine checks from our monitors, you'll never be in the dark about user experience. 
+
+<SideBySide>
+    <Side>
+<img
+    title="Your WPM page"
+    alt=""
+    src={side1wpm}
+/>
+    </Side>
+    <Side>
+<img
+    title="Your WPM page"
+    alt=""
+    src={side2wpm}
+/>    
+    </Side>
+</SideBySide>
+
+<SideBySide>
+    <Side>
+<img
+    title="Your WPM page"
+    alt=""
+    src={side1Bwpm}
+/>
+    </Side>
+    <Side>
+<img
+    title="Your WPM page"
+    alt=""
+    src={side2Bwpm}
+/>     </Side>
+</SideBySide>
+
+## Objectives [#objectives]
+
+This doc helps you set up and understand your data, so you can empower your team to take control of your customer experience. In this tutorial, you will:
+
+* Set up monitoring for two pages with website performance monitoring 
+* Learn about the data collected from WPM
+
+## Set up WPM [#setup]
+<Steps>
+    <Step>
+    ### Get data without installation
+
+    To access WPM, go to **[one.newrelic.com > Website performance monitoring](https://one.newrelic.com/website-grader)**.
+
+    </Step>
+    <Step>
+        ### Choose your URL 
+
+    WPM requires no installation or set up. Once you've chosen pages to monitor, New Relic deploys a set of monitors from different servers around the globe that check each page for website availability, broken links, performance, and SSL certificate expiration. To get started, we recommend choosing these pages:
+
+    * Your homepage.
+    * Pages with critical assets, like important videos, images, or links.
+    * Pages critical to your customer funnel
+
+New Relic will take a few minutes to deploy the monitors, then you'll start receiving data about your website. Keep in mind that WPM doesn't check all pages across your site, only the ones you've manually added.
+
+    </Step>
+    <Step>
+        ### Add Google PageSpeed API key 
+
+When you set up WPM for the first time, New Relic gives you complementary core web vitals via Google PageSpeed. To continue receiving your core web vitals, you need to set up PageSpeed with your own account.
+
+<img
+    title=""
+    alt=""
+    src={pageSpeedAPI}
+/>
+
+    </Step>
+
+    <Step>
+        ### Put your data in context
+
+Once your monitors report data to New Relic, your WPM page displays metrics that can help you improve user experience. WPM gives you four kinds of data:
+
+<CollapserGroup>
+    <Collapser
+        id="ping"
+        title="Check page availability"
+    >
+<img
+    title="View your page availability"
+    alt="View page availability"
+    src={pingWPM}
+/>
+
+The first section of WPM shows you ping data. This kind of data is an availability check, confirming basic site functionalities:
```

Comment:
Also note that, all these metrics are from Desktop view only. They don't give the Mobile view, it is out of scope for WPM.